### DISCUSSION TITLE
[Support] @ApiOperation deprecated 参数无效

### DISCUSSION THREAD

**Discussion by cuifan53** [midwayjs/midway#3430] [Q&A]
### Describe the problem（描述问题）

![image](https://github.com/midwayjs/midway/assets/58646951/c71953bc-fd98-4b20-b581-eca16b2c6061)
在swagger装饰器@ApiOperation中使用了参数deprecated，但是在生成的swagger页面并没有生效

### Midway Versions（Midway 版本）

midway-version@1.3.8

**Comment by czy88840616** [comment]
已知问题，apiOperation 整个好像有点问题。
